technology-overview.md

```diff
@@ -19,6 +19,9 @@ Agoric also works with [ECMA TC53](https://www.ecma-international.org/memento/tc
 - [Realms](https://github.com/tc39/proposal-realms)
 - [Eventual-send operator](https://github.com/Agoric/proposal-infix-bang)
 - [WeakRefs](https://github.com/tc39/proposal-weakrefs/)
+- Jessie, a subset of JavaScript that simplifies writing of deterministic smart contracts
```

Comment:
"determinism" is mostly orthogonal to Jessie vs SES. I would replace with "reliable".